README.md

```diff
@@ -122,6 +122,14 @@ Override border color
 <Avvvatars value="best_user@gmail.com" borderColor="#fff" />
 ```
 
+### `borderRadius?: number`
+
+Override border radius
+
+```jsx
+<Avvvatars value="best_user@gmail.com" borderRadius={10} />
+```
+
```

Comment:
This should be `radius` rather than `borderRadius`, as following the principle from `shadow` and `size`.
User should differentiate borderSize and borderColor is here to represent subset of border, while technically the approach is correct, we want to keep it as simple as possible